lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Model for Course Card.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ], 
+        function (Backbone) {
+        return Backbone.Model.extend({
+            initialize: function(data) {
+                if (data){
+                    this.set({
+                        name: data.display_name,
+                        courseStart: data.course_start,
```

Comment:
I an unsure about converting the keys to camelCase here, it may added unnecessary confusion when the template expects different keys than the API returns.